src/types.ts

```diff
@@ -30,6 +30,7 @@ export type SliderProps = {
     onValueChange?: SliderOnChangeCallback;
     renderAboveThumbComponent?: (index: number) => React.ReactNode;
     renderThumbComponent?: () => React.ReactNode;
+    renderMinimumTrackMarkComponent?: () => React.ReactNode;
```

Comment:
This isn't really rendering a TrackMark vs. the minimum Track itself.  Could you rename this to renderMinimumTrackComponent and remove the `Mark`?

Reply:
Oh... 🤦 I guess I need more coffee 😸